common/models/tags.go

```diff
@@ -59,3 +61,42 @@ func (t Tags) Any(f filter) bool {
 
 	return false
 }
+
+func (t Tags) GroupByID() map[string]Tags {
+	entityTags := map[string]Tags{}
+	for _, tag := range t {
+		tags, ok := entityTags[tag.EntityID]
+		if !ok {
+			tags = Tags{}
+		}
+
+		entityTags[tag.EntityID] = append(tags, tag)
+	}
+
+	return entityTags
+}
+
+// sorting functions
+func (t Tags) Len() int {
+	return len(t)
+}
+
+func (t Tags) Swap(i, j int) {
+	t[i], t[j] = t[j], t[i]
+}
+
+func (t Tags) Less(i, j int) bool {
```

Comment:
Is this needed? I can't see where is is being used?

Reply:
It was in my old tests, but now it's not.